e2e_tests/cli/test_import.py

```diff
@@ -311,3 +374,148 @@ def test_annotation_overwrite_warning(
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        actual_annotations_dir = Path(tmp_dir_str)
+        export_and_download_annotations(
+            actual_annotations_dir, local_dataset, config_values
+        )
+        compare_annotations_export(
+            actual_annotations_dir, expected_annotations_dir, item_type, base_slot
+        )
+
+
+def test_import_annotations_to_multi_channel_item_without_slots_defined(
+    local_dataset: E2EDataset, config_values: ConfigValues
+) -> None:
+    """
+    Upload annotations to a multi-channel item without aligning each annotation to a
+    slot. All annotations should end up the base slot
+    """
+    item_type = "multi_channel"
+    local_dataset.register_read_only_items(config_values, item_type)
+    expected_annotations_dir = (
+        Path(__file__).parents[1]
+        / "data"
+        / "import"
+        / "multi_channel_annotations_without_slots_defined"
+    )
+    result = run_cli_command(
+        f"darwin dataset import {local_dataset.name} darwin {expected_annotations_dir}"
+    )
+    assert_cli(result, 0)
+    base_slot = get_base_slot_of_item(config_values, local_dataset.id, item_idx=0)
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        actual_annotations_dir = Path(tmp_dir_str)
+        export_and_download_annotations(
+            actual_annotations_dir, local_dataset, config_values
+        )
+        compare_annotations_export(
+            actual_annotations_dir, expected_annotations_dir, item_type, base_slot
+        )
+
+
+def test_import_annotations_to_multi_channel_item_with_slots_defined(
+    local_dataset: E2EDataset, config_values: ConfigValues
+) -> None:
+    """
+    Upload annotations to a multi-channel item where each annotation is aligned with
+    the base slot. Each annotation should end up in the base slot
+    """
+    item_type = "multi_channel"
+    local_dataset.register_read_only_items(config_values, item_type="multi_channel")
+    expected_annotations_dir = (
+        Path(__file__).parents[1]
+        / "data"
+        / "import"
+        / "multi_channel_annotations_with_slots_defined"
+    )
+    result = run_cli_command(
+        f"darwin dataset import {local_dataset.name} darwin {expected_annotations_dir}"
+    )
+    assert_cli(result, 0)
+    base_slot = get_base_slot_of_item(config_values, local_dataset.id, item_idx=0)
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        actual_annotations_dir = Path(tmp_dir_str)
+        export_and_download_annotations(
+            actual_annotations_dir, local_dataset, config_values
+        )
+        compare_annotations_export(
+            actual_annotations_dir, expected_annotations_dir, item_type, base_slot
+        )
+
+
+def test_import_annotations_to_multi_channel_item_non_base_slot(
+    local_dataset: E2EDataset, config_values: ConfigValues
+) -> None:
+    """
+    Upload annotations to a multi-channel item where each annotation is aligned with a
```

Comment:
What happens if I have at least one annotation that is aligned to a base_slot? Would that be the only one imported?

Reply:
No - If there is any annotation that is not aligned with the base slot, the importer will throw an error